src/ScssAssetConverter.php

```diff
@@ -25,8 +25,22 @@ class ScssAssetConverter extends Component implements AssetConverterInterface
      * significantly degrade the performance.
      */
     public $forceConvert = false;
-
-    private $compiler;
+    
+    public $formatter = null;
+    
+    protected $scssPhpFormatterMap =
+    [
+        'compact'       => '\Leafo\ScssPhp\Formatter\Compact',
+        'compressed'    => '\Leafo\ScssPhp\Formatter\Compressed',
+        'crunched'      => '\Leafo\ScssPhp\Formatter\Crunched',
+        'debug'         => '\Leafo\ScssPhp\Formatter\Debug',
+        'expanded'      => '\Leafo\ScssPhp\Formatter\Expanded',
+        'nested'        => '\Leafo\ScssPhp\Formatter\Nested',
+    ];
+
+    protected $defaultFormatter = 'nested';
```

Comment:
Why not set `protected $formatter = 'nested';` instead? This variable (`$defaultFormatter`) is not needed.

Reply:
Agree with you, I have removed defaultFormatter